sig-docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing to SIG Docs
+
+SIG Docs focuses on creating, maintaining, and publishing documentation for the Kubernetes project
+in the kubernetes.io [website](https://github.com/kubernetes/website), 
```

Comment:
Shouldn't this link to the actual website and the below link to the actual blog instead of their GH Repo?

Reply:
Updated links